src/server.c

```diff
@@ -235,43 +235,41 @@ void serverServe(Server *server)
 
     int sock = makeSocket(server->port);
     int newSock;
+    int fd_size = sock + 1;
```

Comment:
The naming of `fd_size` is a bit misleading. Essentially speaking, `fd_size` refers to file descriptor rather than the length.